rottentomatoes/standalone.py

```diff
@@ -83,7 +83,10 @@ def _get_score_details(content: str) -> object:
 
     tomatometer_score = int(soup.find('rt-button', {'slot': 'criticsScore'}).text.strip("%\n"))
     audience_score = int(soup.find('rt-button', {'slot': 'audienceScore'}).text.strip("%\n"))
-    rating = soup.find('rt-text', {'slot': 'ratingsCode'}).text
+    try:
+        rating = soup.find('rt-text', {'slot': 'ratingsCode'}).text
+    except:
```

Comment:
Can you catch this error with a more specific `Exception` to prevent unexpected handling, i.e. with `KeyboardInterrupt`?